src/libaktualizr/storage/sqlstorage.cc

```diff
@@ -1397,7 +1397,10 @@ bool SQLStorage::loadInstalledVersions(const std::string& ecu_serial, boost::opt
 bool SQLStorage::hasPendingInstall() {
   SQLite3Guard db = dbConnection();
 
-  auto statement = db.prepareStatement("SELECT count(*) FROM installed_versions where is_pending = 1");
+  auto statement = db.prepareStatement(
```

Comment:
The business logic in SotaUptaneClient::fetchMeta() expects that hasPendingInstall/hasPendingUpdates returns true if there is at least one pending update regardless if it's Primary or Secondary.